docs/manualGrading/index.md

```diff
@@ -94,10 +94,61 @@ The user interface for manual grading is built to encourage a workflow where all
+
+Each rubric item is set a number of points and a description. This description is shown to both graders and in feedback to students. It is also possible to add an additional detailed explanation to the rubric item, which is shown to students upon request. Finally, a grader note can be set that is visible to graders, but not to students, and can be used to provide instructions on scenarios when the rubric should be used or not.
+
+Items can also be set to always be listed for students (default), or only if it is is selected. This option may be used in cases such as penalties that apply to a small subset of students, or for credit assigned to unconventional correct answers that are not to be advertised to students that did not use them.
+
+In the description, explanation, and grader note, it is possible to use mustache patterns based on the parameters, correct answers and submitted answers for a submission. This will cause the description of the item provided to graders and students to match the question variant assigned to the student, as shown in the screenshot below.
+
+![Rubric items in settings](rubric_items_setting.png)
+
+This will result in a grading panel containing a description with the value of the parameter replaced with the appropriate value for the student's variant.
+
+![Rubric items in grading panel](grading_panel_with_rubrics.png)
+
+Student's feedback is also replaced in a similar manner.
+
+![Rubric items in student feedback](student_feedback_with_rubrics.png)
+
+As grading progresses, new scenarios may be found by graders that require changes in the rubric, such as new types of penalties or credits or specific kinds of feedback. The rubric may be changed as required, even after grading is already underway. These changes may affect existing submissions:
+
+- Adding new rubric items does not affect the score of existing submissions;
+- Changing the points assigned to individual rubric items will cause any existing submission gradings that use the rubric item to be recalculated with the new points assigned to the items;
+- Changing the description or explanation of individual rubric items will update the feedback provided to students, but not the score;
+- Deleting rubric items will cause existing gradings that use the rubric item to be recalculated without the item;
+- Changing the minimum score or maximum extra credit will cause all existing rubric gradings to be recalculated.
+
+When updating rubric settings, a flag at the bottom of the settings dialog box gives graders the option of tagging all submissions for manual grading. This option may be used in cases where the changes are significant enough that they warrant a review of all previously graded questions to ensure they take the new rubric into account. This is encouraged in cases where rubric items applied to multiple rubric gradings are deleted.
```

Comment:
```suggestion
When updating rubric settings, a flag at the bottom of the settings dialog box gives graders the option of tagging all submissions (with that rubric item applied) for manual grading. This option may be used in cases where the changes are significant enough to warrant a review of all previously graded questions to ensure the new rubric is taken into account. Substantial changes to rubric item descriptions would normally require at least a spot-check of previously graded submissions.
```

That last sentence was confusing to me...consider rephrasing or removing?

Reply:
Your sentence is incorrect. Tagging all submissions for new grading applies to all submissions, not only to those with the items changed.

```suggestion
When updating rubric settings, a flag at the bottom of the settings dialog box gives graders the option of tagging all submissions for manual grading. This option may be used in cases where the changes are significant enough to warrant a review of all previously graded questions to ensure they take the new rubric into account. This is encouraged in cases where substantial changes are made, which may require at least a spot-check of previously graded submissions.
```